### PR TITLE
Update messages_de.properties

### PR DESCRIPTION
**Pull request by ch-lepp**
Added missing German translations

### CODE REVIEW

**Comment by jonkoops** [comment]
Could you [sign off](https://github.com/keycloak/keycloak/pull/28962/checks?check_run_id=24094141686) on your commit?

**Comment by ch-lepp** [comment]
Done, I think...

**Comment by edewit** [comment]
@ch-lepp you can do a `git rebase upstream/main --signoff`

**Comment by ch-lepp** [comment]
Ok, I think I've got it now.

Thx

**Comment by jonkoops** [comment]
@ch-lepp if you can get another German native speaker to review your work we can merge this in.

**Comment by jonkoops** [comment]
Don't worry about the CI failures, we're working on a fix for it.

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties [hunk 1]
```diff
@@ -199,6 +228,28 @@ missingTotpMessage=Bitte geben Sie den One-time Code ein.
 missingTotpDeviceNameMessage=Bitte geben Sie einen Gerätenamen ein.
 notMatchPasswordMessage=Passwörter sind nicht identisch.
 
+error-invalid-value=Ungültiger Wert.
+error-invalid-blank=Bitte geben Sie einen Wert an.
+error-empty=Bitte geben Sie einen Wert an.
+error-invalid-length=Länge muss zwischen {1} und {2} liegen.
```
Comment (Persi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:234]: Vorschlag: Länge muss zwischen {1} und {2} Zeichen liegen.

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties [hunk 2]
```diff
@@ -199,6 +228,28 @@ missingTotpMessage=Bitte geben Sie den One-time Code ein.
...
+error-invalid-length-too-short=Minimale Länge ist {1}.
+error-invalid-length-too-long=Maximale Länge ist {2}.
+error-invalid-email=Ungültige E-Mail_Adresse.
```
Comment (Persi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:237]: E-Mail-Adresse

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties [hunk 3]
```diff
@@ -199,6 +228,28 @@ missingTotpMessage=Bitte geben Sie den One-time Code ein.
...
+error-pattern-no-match=Ungültiger Wert.
+error-invalid-uri=Ungültige URL.
+error-invalid-uri-scheme=Ungültiges URL-Schema.
+error-invalid-uri-fragment=Ungültiger URL-Bestandteil.
+error-user-attribute-required=Bitte füllen Sie dieses Feld aus.
+error-invalid-date=Ungültiges Datum.
+error-user-attribute-read-only=Dieses Feld darf nicht editiert werden.
+error-username-invalid-character=Wert enthält ungültiges Zeichen.
+error-person-name-invalid-character=Wert enthält ungültiges Zeichen.
+error-reset-otp-missing-id=Bitte wählen Sie eine OTP Konfiguration aus.
```
Comment (Persi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:251]: OTP-Konfiguration

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties [hunk 4]
```diff
@@ -219,8 +270,10 @@ confirmLinkIdpReviewProfile=Benutzerkonto überprüfen
 confirmLinkIdpContinue=Zu einem bestehenden Benutzerkonto hinzufügen
 
 configureTotpMessage=Sie müssen eine Mehrfachauthentifizierung einrichten, um das Benutzerkonto zu aktivieren.
+configureBackupCodesMessage=Sie müssen Backup Codes einrichten, um das Benutzerkonto zu aktivieren.
 updateProfileMessage=Sie müssen Ihr Benutzerkonto aktualisieren, um das Benutzerkonto zu aktivieren.
 updatePasswordMessage=Sie müssen Ihr Passwort ändern, um das Benutzerkonto zu aktivieren.
+updateEmailMessage=Sie müssen Ihre Email-Adresse aktualisieren, um das Benutzerkonto zu aktivieren..
```
Comment (Persi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:276]: E-Mail-Adresse

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties [hunk 5]
```diff
@@ -280,15 +334,20 @@ invalidAccessCodeMessage=Ungültiger Access-Code.
 sessionNotActiveMessage=Session nicht aktiv.
 invalidCodeMessage=Ungültiger Code, bitte melden Sie sich erneut über die Applikation an.
 cookieNotFoundMessage=Cookie konnte nicht gefunden werden. Bitte stellen Sie sicher, dass Cookies in Ihrem Browser aktiviert sind.
+insufficientLevelOfAuthentication=Die angefragte Sträke des Authentifizierungsverfahrens wurde nicht erfüllt.
```
Comment (Persi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:337]: Stärke

File: themes/src/main/resources-community/theme/base/login/messages/messages_de.properties [hunk 6]
```diff
@@ -280,15 +334,20 @@ invalidAccessCodeMessage=Ungültiger Access-Code.
...
 identityProviderUnexpectedErrorMessage=Unerwarteter Fehler während der Authentifizierung mit dem Identity Provider.
 identityProviderMissingStateMessage=Fehlender state Parameter in der Antwort vom Identitätsanbieter.
+identityProviderMissingCodeOrErrorMessage=Fehlender code oder error Parameter in der Antwort vom Identitätsanbieter.
```
Comment (Persi) [inline themes/src/main/resources-community/theme/base/login/messages/messages_de.properties:340]: Fehlender Code- oder Fehlerparameter in der Antwort vom Identitätsanbieter.

**Review by Persi — CHANGES_REQUESTED** [review]
I've commented some suggestions.

**Comment by ch-lepp** [comment]
I applied the review suggestions

**Review by Persi — APPROVED** [review]
lgtm

**Comment by jonkoops** [comment]
Thanks all, I will set the PR to merge.

**Comment by jonkoops** [comment]
Please refrain from merging things in from `main` unless absolutely necessary, as this causes CI to re-trigger and creates additional work for us.

**Review by ahus1 — APPROVED** [review]

**Pull request merged**